### ISSUE TITLE
[Quest] Skywing

### ISSUE THREAD

**Issue by stealer0517** [WoWManiaUK/Redemption#5283]
https://www.wowhead.com/quest=10898/skywing

Once you accept the quest Skywing just stays in place and never starts moving to his destination.
After abandoning the quest I can't re get the quest from him so I think he's stuck in "in a quest" mode. 

I tried waiting for about 5 minutes last night and Skywing never moved so I abandoned it and couldn't get it again.
I came back a few hours later and again almost 24 hours later and still couldn't get the quest from Skywing again.
https://i.imgur.com/Pcuu0fj.png

**Comment by Greywolf59** [comment]
Sometimes this one works, sometimes nor. No clue on what is it based. Maybe because someone before you fails or abandons. I have kited the mobs to kill him on spawn point to reset his quest once iirc

**Comment by svijetlo** [comment]
this one is strange , but it did work to me every time i did it on both our server

**Comment by Rushor** [comment]
i think we had a similar issue with harrison jones.

so somtimes quest escort creatures simply will not start pathing right?

**Comment by Rushor** [comment]
I have an idea.

Currently we have that 500 ms delay between thr path points due to Performance improve. 

I will remove that for Escort creatures and then we will see

**Comment by jeyrella** [comment]
This is an odd one.  We tried to fix this last year in #5093 and testing at that time worked flawlessly after the last changes.  You're right rushor that this has hallmarks of the issue with Harrison Jones.  

WIll try it again tonight, but as in previous testing  it worked fine (i.e. the issue was not reproducible), testing now won't show anything other than "it isn't completely broken now"...

**Comment by Rushor** [comment]
Yeah

**Comment by svijetlo** [comment]
![WoWScrnShot_020521_103244](https://user-images.githubusercontent.com/61908373/107022961-4680a480-67a6-11eb-9577-565d07a50220.jpg)
![WoWScrnShot_020521_103316](https://user-images.githubusercontent.com/61908373/107022969-497b9500-67a6-11eb-8757-909598f7a8ee.jpg)
![WoWScrnShot_020521_103336](https://user-images.githubusercontent.com/61908373/107022976-4b455880-67a6-11eb-99b3-3019dcb41ca5.jpg)
![WoWScrnShot_020521_103450](https://user-images.githubusercontent.com/61908373/107022984-4da7b280-67a6-11eb-8de9-6d8fbc97c93b.jpg)
![WoWScrnShot_020521_103512](https://user-images.githubusercontent.com/61908373/107022989-500a0c80-67a6-11eb-87ae-ecb08fbbe6ca.jpg)
![WoWScrnShot_020521_104447](https://user-images.githubusercontent.com/61908373/107022999-5304fd00-67a6-11eb-8518-26734af08da8.jpg)
![Uploading WoWScrnShot_020521_113525.jpg…]()

**Comment by svijetlo** [comment]
completed quest ones normal way -work
did it 2 times by talking to it from max talk distance -work 
let if fly alone to fail , came back 30 mins later - he was back - work 
killed it - came back cca 30 min later - he was there - work 
SS form above show some things

**Comment by Rushor** [comment]
very good!

this fix also makes all escort creatures, db escorties like skywing and all creature with waypoint movement look smooth again.

Before they have been stopped at every path for 500 ms

**Comment by jeyrella** [comment]
reopening from #5336

**Comment by Rushor** [comment]
can someone test on ptr again?

**Comment by wow-mania** [comment]
@Rushor please do not change general code that affects the entire server without informing me. Also for these cases, we need a different type of testing and testers will need to spend a long time on it.

**Comment by jeyrella** [comment]
Sigh.  I couldn't replicate this on PTR.  seems to work perfectly.  How annoying

**Comment by svijetlo** [comment]
ok ok ok smt is strange here i just try this quest on ptr and it worked perfect for me will add SS of his path etc

**Comment by svijetlo** [comment]
![WoWScrnShot_030721_153920](https://user-images.githubusercontent.com/61908373/110243675-b035cf00-7f5b-11eb-92e3-b0c0a324a230.jpg)
![WoWScrnShot_030721_153949](https://user-images.githubusercontent.com/61908373/110243679-b2982900-7f5b-11eb-9b5e-d17bc135fd22.jpg)
![WoWScrnShot_030721_154021](https://user-images.githubusercontent.com/61908373/110243681-b461ec80-7f5b-11eb-8382-36cb531d3f5f.jpg)
![WoWScrnShot_030721_154028](https://user-images.githubusercontent.com/61908373/110243682-b6c44680-7f5b-11eb-9c94-f185901c0361.jpg)
![WoWScrnShot_030721_154030](https://user-images.githubusercontent.com/61908373/110243696-bd52be00-7f5b-11eb-9aa3-1baa5ed9d29c.jpg)
![WoWScrnShot_030721_154037](https://user-images.githubusercontent.com/61908373/110243699-bfb51800-7f5b-11eb-9672-40358d229213.jpg)
![WoWScrnShot_030721_153820](https://user-images.githubusercontent.com/61908373/110243702-c2177200-7f5b-11eb-893d-a0a0df68dca2.jpg)
![WoWScrnShot_030721_153829](https://user-images.githubusercontent.com/61908373/110243704-c3e13580-7f5b-11eb-9f61-b71263a505f4.jpg)
![WoWScrnShot_030721_153839](https://user-images.githubusercontent.com/61908373/110243706-c5aaf900-7f5b-11eb-98e1-87a4bbc5d9e7.jpg)
![WoWScrnShot_030721_153844](https://user-images.githubusercontent.com/61908373/110243707-c774bc80-7f5b-11eb-88ba-5970e86fc1d1.jpg)
![WoWScrnShot_030721_153859](https://user-images.githubusercontent.com/61908373/110243709-c8a5e980-7f5b-11eb-8d0b-d644d0882ccb.jpg)

**Comment by jeyrella** [comment]
yup.  same here.  i tried abandoning & retaking, completing and coming back....  just can't reproduce this at all.  BUT... I have seen it in the past, so I can believe the reports it is still occurring

**Comment by Rushor** [comment]
Leave this as is. It work totally fine for all other Escort and my fix was totally valid.

So if there is anything borked it must be this Individual npc by itself.

**Comment by jeyrella** [comment]
Ok.  then unfortunately this one is as-is.

**Comment by jeyrella** [comment]
for notes, just completed this in live, and worked flawlessly there too.

**Comment by rnel76** [comment]
any chance this quest could be made autocomplete or something? for those of us who try and get loremaster

**Comment by jeyrella** [comment]
This is not needed for Loremaster.  There are 139 quests that count to _Terror of Terrokar_, only 68 of those are needed for Loremaster, and this standalone quest does not gate out any other quests.  Also, it will reset naturally on server restarts.

**Comment by rnel76** [comment]
First, thank you for your reply. I'm aware it is not needed. But it's a fun
little quest, with a nice reward and rep. I realise that for the most part,
questing is not what most people care, about that play. It's leveling, go
hfp, maybe zangermarsh, than nagrand and gone. But I am part of that money
who quest for the sheer joy of questing. Why should I, and and however many
other players that are like me be forced to pay the price for what seems
like bad coding in blizzards end?
I made a ticket the same day I made the comment, and it was resolved with
the GM killing skywing, with respawn it worked just fine. But with only 2
tickets open, it still took over an hour, and close to if not a half hour
of it was time when my ticket was the only open ticket. I know, tickets
aren't a GM's responsibility. I mention this for that exact reason, is it
right to waste the GM's time everytime I go after terror of terrokar? I
really do appreciate the effort everyone puts into the server, and I
understand somethings may not be able to be fixed. But if the server is
trinity based, there are work arounds. Don't make players pay for what
can't be fixed.
